lib/Sema/TypeCheckType.cpp

```diff
@@ -2221,8 +2221,21 @@ Type TypeResolver::resolveAttributedType(TypeAttributes &attrs,
 
     for (auto i : FunctionAttrs) {
       if (attrs.has(i)) {
-        TC.diagnose(attrs.getLoc(i), diag::attribute_requires_function_type,
-                    TypeAttributes::getAttrName(i));
+          if (attrs.has(TAK_escaping)) {
```

Comment:
Noted. I use four space for my app projects. Any pro tip for quickly switching back and forth depending on the project?